### PR TITLE
removing conductor from docs and nav

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-72807

## Description of changes being made
Removing Conductor from docs and nav, le sigh.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by pierrebeitz — COMMENTED** [review]
@msdolbey i added three commits. the first one is kind of mandatory, as else we'll have dead links in e.g. the navbars or the main landing page.

then there are two further commits that i'm not 100% sure about.  
one removes a section that mentions conductor-training and the other one removes a section with some `TODO` in it. 🤷‍♂️ 
feel free to alter or remove any of those commits.

**Review by pierrebeitz — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
Some changes to Konvoy, so we need to backport this.

File: pages/ksphere/konvoy/1.6/training/index.md [hunk 1]
```diff
@@ -18,10 +18,3 @@ D2IQ provides training for all of the D2IQ products. Two types of training are a
   - Live instructor for Q&A
   - Content for intermediate and advanced learners
 
-- [Training using Conductor](https://d2iq.com/products/conductor)
```
Comment (cneth) [inline pages/ksphere/konvoy/1.6/training/index.md:21]: You need to also adjust line 13 in this page as it says "Two types of training" but now there will only be one section.   

This page is in the Konvoy repo; I don't understand how the Konvoy docs are merged into this repo so I am not sure you need to change this here.  If you do, note that there is already a Konvoy PR that touches this page:

https://github.com/mesosphere/konvoy/pull/2367

**Review by cneth — CHANGES_REQUESTED** [review]

File: layouts/partials/head.pug [hunk 2]
```diff
@@ -48,6 +48,6 @@ if env != 'development'
   script.
     (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
     new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src
```
[the remark this answers is not in the document]
Comment (pierrebeitz) [reply layouts/partials/head.pug]: that's insignificant. let's get it out of this PR nonetheless.

**Review by cneth — APPROVED** [review]

File: pages/ksphere/konvoy/1.6/training/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/ksphere/konvoy/1.6/training/index.md:21]: @cneth @pierrebeitz - I'll push a commit for this, because we're already modifying these pages.

**Review by msdolbey — COMMENTED** [review]
Thanks for the updates, @pierrebeitz 👍 

sad conductor

**Comment by cmastr** [comment]
cannot go forward with this at this time.

**Pull request closed without merging**